GCRCatalogs/register.py

```diff
@@ -558,12 +582,14 @@ def get_root_dir():
     return _config_register.root_dir
 
 
-def set_root_dir(path):
+def set_root_dir(path, write_to_config=False):
     """
     Sets runtime root_dir to *path*.
     """
     _config_register.root_dir = path
 
+    if write_to_config:
+        _config_register.persist_root_dir()
```

Comment:
Two blank lines between module-level functions.

```suggestion
        _config_register.persist_root_dir()

```